docs/en/latest/configuration/config-default.md

```diff
@@ -0,0 +1,1028 @@
+---
+title: Configuration Options
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This page describes the configuration options defined in the `conf/config-default.yaml`.
+
+## `apisix`
+
+Configuration options for `apisix`.
+
+### `admin_api_mtls`
+
+```yaml
+# Depends on `port_admin` and `https_admin`.
+admin_api_mtls:
+
+  # (string) Path of your self-signed server side cert.
+  admin_ssl_cert: ""
+
+  # (string) Path of your self-signed server side key.
+  admin_ssl_cert_key: ""
+
+  # (string) Path of your self-signed ca cert.The CA is used to sign all admin api callers' certificates.
+  admin_ssl_ca_cert: ""
+```
+
+### `admin_listen`
```

Comment:
Could you update this PR according to the latest https://github.com/apache/apisix/blob/master/conf/config-default.yaml?

We have just made a big refactor for 3.0.0

Reply:
Sure let me update